src/main/java/com/github/onsdigital/babbage/search/external/requests/search/requests/ContentQuery.java

```diff
@@ -96,4 +97,35 @@ protected ContentResponse getContentResponse() throws Exception {
                 .content(new StringContentProvider(MAPPER.writeValueAsString(content)));
         return request.send();
     }
+
+    @Override
+    public SearchResult call() throws Exception {
+        SearchResult result = super.call();
+
+        if (Configuration.SEARCH_SERVICE.EXTERNAL_SPELLCHECK_ENABLED) {
+            List<SpellingCorrection> corrections = this.getSpellingCorrections();
+
+            // Build the suggested correction string
+            String suggestedCorrection = SpellCheckRequest.buildSuggestedCorrection(this.searchTerm, corrections,
+                    Configuration.SEARCH_SERVICE.SPELL_CHECK_CONFIDENCE_THRESHOLD);
+
+            // Set the suggestions
+            List<String> suggestions = result.getSuggestions();
+            if (null != suggestions) {
+                suggestions.add(0, suggestedCorrection);
+            } else {
+                suggestions = Collections.singletonList(suggestedCorrection);
+            }
+            result.setSuggestions(suggestions);
+        }
+
+        return result;
+    }
+
+    public List<SpellingCorrection> getSpellingCorrections() throws Exception {
+        SpellCheckRequest spellCheckRequest = new SpellCheckRequest(super.searchTerm);
+        List<SpellingCorrection> corrections = spellCheckRequest.call();
```

Comment:
Don't need to assign here can just `return spellCheckRequest.call();` while this is also picky its 1 less object reference to maintain for the JVM and with my "performance hat" if you aren't doing anything with the ref other than returning it you can just return.